packages/sdk/src/index.ts

```diff
@@ -259,6 +259,9 @@ export class WHIPClient extends EventEmitter {
 
     if (response.ok) {
       this.resource = response.headers.get("Location");
+      if (!this.resource.match(/^http/)) {
+        this.resource = this.whipEndpoint.protocol + "//" + this.whipEndpoint.host + this.resource;
```

Comment:
Perhaps  `this.resource = new URL(this.resource, this.whipEndpoint)`? Seems like you can use that regardless if it's relative or absolute. If it's absolute the base (this.whipEndpoint) will be ignored.

Reply:
( that way we avoid any weirdness if the relative url doesn't start with `/`

Reply:
Hmm, but what if `this.whipEndpoint` is `http://hostname:port/hej/hopp`? Will it not be come `http://hostname:port/hej/hopp` + (the relative url)

Reply:
![Screenshot 2023-02-20 at 12 22 37](https://user-images.githubusercontent.com/4589630/220092500-f45f1bb5-a984-416a-9b42-64c5f9d9c0dc.png)

This is the behavior

Reply:
Not sure what would be desired to be honest 😬

Reply:
That behavior is what we want I think. Thanks for verifying. Will update the PR